diffpy/pdfmorph/pdfplot.py

```diff
@@ -81,6 +84,8 @@ def comparePDFs(pairlist, labels=None, rmin = None, rmax = None, show = True,
     line.  The difference curve will be in green and offset for clarity.
     
     """
+    plt.style.use(bg_mpl_style)
```

Comment:
I don't think this is needed here.   When you set the plot-style it is set globally.  Remove and see if it still works as expected.

Reply:
Removed, thought it might be needed for each of the functions because I didn't write it the same way the wiki says to set it as the default. Tested and it works fine.